### DISCUSSION TITLE
[Question] What is the best for privacy/security, DNScrypt with Relays, or DNS Proxy Orbot?

### DISCUSSION THREAD

**Discussion by ghost** [celzero/rethink-app#1108] [Q&A] [answered]
As the name says,
I got a dilemma what to choose.
What are the pros and cons of DNScrypt with Relays vs. DNS Proxy Orbot (localhost:5400)?

**Comment by ignoramous** [comment, marked as answer]
DNS over Orbot is neat, but I believe DNSCrypt+Relays achieves almost similar security but at a much lower latency. Upcoming release, `v055b`, adds support for [*Oblivious DNS-over-HTTPS*](https://datatracker.ietf.org/doc/rfc9230/), which is similar to DNSCrypt+Relays, too.
